src/sparsezoo/utils/onnx/analysis.py

```diff
@@ -385,13 +385,16 @@ def get_node_weight(
     :param node: node to which parameter belongs to
     :return: a numpy array of param value, None if not found
     """
-
     initializer_name = get_node_weight_name(model_graph, node)
     weight = get_initializer_value(model_graph, node, initializer_name)
+
     if initializer_name is not None and weight is None and node.op_type != "Gather":
         raise Exception(f"Parameter for {node.name} not found")
 
-    return weight
+    # some weights are not accessible, and returns the zero_points, which are scalars
+    if weight is not None and weight.size > 1:
```

Comment:
for forward facing support with channel-wise would be good to have the condition be `.dim > 1` instead of size (channelwise scales will be vector with 1 dimension)